packages/mermaid/src/diagrams/state/parser/stateDiagram.jison

```diff
@@ -65,14 +65,14 @@
 \%%[^\n]*                        /* skip comments */
 "scale"\s+            { this.pushState('SCALE'); /* console.log('Got scale', yytext);*/ return 'scale'; }
 <SCALE>\d+            return 'WIDTH';
-<SCALE>\s+"width"     {this.popState();}
+<SCALE>\s+"width"     {this.popState(); }
```

Comment:
This change (in the .jison file) was just for readability. IMNSHO, having a space before the closing "}" makes it easier to see the final ";" of the last statement; it makes it easier to read.

It can be reverted.

Reply:
```suggestion
<SCALE>\s+"width"     { this.popState(); }
```

My suggestion was to keep it symmetrical, I'm all for extra spaces and readability :)